.gitignore

```diff
@@ -2,4 +2,5 @@
 /app/build
 /.idea
 /.gradle
-*.iml
\ No newline at end of file
+*.iml
+gradle.properties
```

Comment:
That's a bit extreme, now anyone who checks this out will have to add the gradle.properties file with the useAndroidX and whatnot in it. Better to simply not commit the code for accessing our private maven repo in `build.gradle`.

Reply:
fixed it